envbuilder.go

```diff
@@ -269,25 +99,26 @@ func Run(ctx context.Context, options Options) error {
 	}
 	if options.WorkspaceFolder == "" {
 		var err error
-		options.WorkspaceFolder, err = DefaultWorkspaceFolder(options.GitURL)
+		folder, err := DefaultWorkspaceFolder(options.GitURL)
+		options.WorkspaceFolder = folder
```

Comment:
nit: assigning `options.WorkspaceFolder` should probably go after `return err`, and you can probably also remove the declaration of `err` on line 101.